src/service/navigation.js

```diff
@@ -279,8 +284,8 @@ export class Navigation {
     }
 
     // Handle anchor transformations.
-    this.anchorMutators_.forEach(callback => {
-      callback(target);
+    this.anchorMutators_.forEach(anchorMutator => {
+      anchorMutator(target);
       location = this.parseUrl_(target.href);
```

Comment:
You can do this just once after the `forEach`, then.

Reply:
done! thanks